### PR TITLE
Answer

### PR DESCRIPTION
**Pull request by HWLOU**

### CODE REVIEW

File: a1_18.m [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+clear;
+syms Qm Qc elta xi
+Qc = 1;
+elta = 0.35;
```
Comment (piandi) [inline a1_18.m]: change "elta" into "eta"

File: a1_18.m [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+xi = 3;
+W = Qc/xi;
+Qm = W/elta;
+disp(sprintf('���ȼ�����Ϊ %.3f kJ \n', double(Qm)))
```
Comment (piandi) [inline a1_18.m]: Use fprintf() due to disp(sprintf()) is out dated.

File: a2_10.m [hunk 3]
```diff
@@ -0,0 +1,18 @@
+clear;
+syms P1 v1 P2 v2  q qm w deltau;
+x={100,0.845,800,0.175,139,-50,10};
+[P1,v1,P2,v2,deltau,q,c]=deal(x{:});
```
Comment (piandi) [inline a2_10.m]: Directly assign the values into the variables.

File: a2_18.m [hunk 4]
```diff
@@ -0,0 +1,19 @@
+clear;
+syms dQ dE_CV h_1 h_2 c_1 c_2 z_1 z_2 dm_1 dm_2 dW_i;
+dQ=0;dW_i=0;dm_2=0;c_f1=0;z_2=0;z_1=0;
+eq=dE_CV==dQ-(dm_2*(h_2+0.5*c_2^2+9.8*z_2)-dm_1*(h_1+0.5*c_1^2+9.8*z_1)+dW_i);
+[dE_CV]=solve(eq)
```
Comment (piandi) [inline a2_18.m]: Declare the solving variable. It is risky to let Matlab guess what you want.

**Review by piandi — CHANGES_REQUESTED** [review]

**Comment by HWLOU** [comment]
Please send me students'chapter 3 homework.

**Comment by piandi** [comment]
@HWLOU  在所有你写的代码前两行注释下你的姓名首字母缩写，例如% by LHW on 2019-5-4

**Pull request closed without merging**